### PR TITLE
Creating a second Poll in Meetings will overvwrite the first one.

### PR DESCRIPTION
**Pull request by CostiChD**
Issue: https://github.com/xwikisas/application-meeting/issues/27

### CODE REVIEW

File: ui/src/main/resources/Meeting/Code/MeetingSheet.xml [hunk 1]
```diff
@@ -220,7 +225,12 @@
                 &lt;/div&gt;
               &lt;/form&gt;
             #else
-              #set($url = $xwiki.getURL("XPoll.XPoll for the Meeting $doc.name"))
+              #set($meetingName = $doc.name)
+              #set($defaultDocName = $services.model.getEntityReference('DOCUMENT', 'default').name)
+              #if($meetingName == $defaultDocName)
+                #set($meetingName = $doc.documentReference.lastSpaceReference.name)
+              #end
```
Comment (acotiuga) [inline ui/src/main/resources/Meeting/Code/MeetingSheet.xml]: You could extract the duplicated code in a macro that returns the poll title.

Comment (CostiChD) [reply ui/src/main/resources/Meeting/Code/MeetingSheet.xml]: Yes, I forgot that.

File: ui/src/main/resources/Meeting/Code/MeetingSheet.xml [hunk 2]
```diff
@@ -52,6 +52,14 @@
   : $doc.display($propName)
   #end
 #end
+#macro (setTitle)
```
Comment (acotiuga) [inline ui/src/main/resources/Meeting/Code/MeetingSheet.xml]: You should be more specific here and name the macro something like `setPollTitle` and also rename accordingly the `$title` variable defined at the end of the macro.

File: ui/src/main/resources/Meeting/Code/MeetingSheet.xml [hunk 3]
```diff
@@ -52,6 +52,14 @@
   : $doc.display($propName)
   #end
 #end
+#macro (setPollTitle)
+  #set($meetingName = $doc.name)
+  #set($defaultDocName = $services.model.getEntityReference('DOCUMENT', 'default').name)
+  #if($meetingName == $defaultDocName)
+    #set($meetingName = $doc.documentReference.lastSpaceReference.name)
+  #end
+  #set($pollName="XPoll for the Meeting $meetingName")
```
Comment (acotiuga) [inline ui/src/main/resources/Meeting/Code/MeetingSheet.xml]: Well, if you call this `$pollName`, then the macro should set the poll name and not the poll title, right?

Comment (CostiChD) [reply ui/src/main/resources/Meeting/Code/MeetingSheet.xml]: Changing it right now.

File: ui/src/main/resources/Meeting/Code/MeetingSheet.xml [hunk 4]
```diff
@@ -205,7 +205,9 @@
       (((
         (% class="col-sm-12 col-xs-12" %)
         (((
-          #if($xcontext.action == 'view' &amp;&amp; $doc.getValue('displayMap') == 1 &amp;&amp; "${doc.getValue('place')}" != '')
+          #if($xcontext.action == 'view'
+              &amp;&amp; $doc.getValue('displayMap') == 1
+              &amp;&amp; "${doc.getValue('place')}" != '')
```
Comment (acotiuga) [inline ui/src/main/resources/Meeting/Code/MeetingSheet.xml]: If it's enough space in a line, I would prefer to use it :) There's no need to have new line for each condition.

Comment (CostiChD) [reply ui/src/main/resources/Meeting/Code/MeetingSheet.xml]: Changing it in a moment.

**Pull request merged**